### PR TITLE
Feat/nfts more events

### PR DESCRIPTION
**Pull request by rsteubs-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts [hunk 1]
```diff
@@ -222,6 +222,11 @@ export default ({ api }: { api: APIType }) => {
     }
   }
 
+  const getAmountUsd = function* (coin: string, amount: number) {
+    const usdPrice = yield call(api.getPriceIndex, coin, 'USD', new Date().getTime())
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts:226]: typescript

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Status/index.tsx [hunk 2]
```diff
@@ -143,6 +150,25 @@ const NftOrderStatus: React.FC<Props> = (props) => {
           <SpinningLoader height='14px' width='14px' borderWidth='3px' />
         </Wrapper>
       )}
+      {props.orderFlow.status === NftOrderStatusEnum.READY_FOR_SALE && (
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Status/index.tsx:153]: small thing but READY_FOR_SALE sounds like it is ready for sale not loading.

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Status/index.tsx [hunk 3]
```diff
@@ -213,13 +239,14 @@ const NftOrderStatus: React.FC<Props> = (props) => {
               data-e2e='returnToMarketPlace'
             >
               <FormattedMessage
-                id='buttons.close_and_view_item'
+                id='buttons.return_to_marketplace'
                 defaultMessage='Close and View Item'
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Status/index.tsx:243]: ?

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Status/index.tsx (hunk 2 above)
Comment (rsteubs-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Status/index.tsx:152]: yeah, should be POST_LISTING actually, thanks

**Review by plondon — APPROVED** [review]

**Pull request merged**